src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/MultipleRunsPerSarifTests.cs

```diff
@@ -75,7 +73,7 @@ public ErrorListTests()
                 }
             };
 
-            ErrorListService.ProcessSarifLog(testLog, "TestPath.sarif", null);
+            TestUtilities.InitializeTestEnvironment(testLog);
```

Comment:
>InitializeTestEnvironment [](start = 26, length = 25)

Seems like this helper should accept the file path arg?

Reply:
Not sure. I don't actually use a file called "TestPath.sarif", the only reason that's there is because the ProcessSarifLog takes it in. I don't use the value in any tests. Is there value in having this specified, or should I just change it to "" instead?